### DISCUSSION TITLE
Updates to project depencencies

### DISCUSSION THREAD

**Discussion by kvet** [react-static/react-static#1532] [Q&A] [answered]
Hi. Right now many packages used in react-static are upgraded with the new major versions. Do you have plans to upgrade to react@17 & webpack@5 at least?

**Comment by SleeplessByte** [comment, marked as answer]
Both might require significant work. Happy to take PRs, but there is currently no longer financial backing to update this regularly (cuzzz COVID bankrupted the paying clients).
